src/uct/sm/mm/mm_md.c

```diff
@@ -34,6 +34,8 @@ ucs_status_t uct_mm_mem_alloc(uct_md_h md, size_t *length_p, void **address_p,
         return UCS_ERR_NO_MEMORY;
     }
 
+    seg->address = (flags & UCT_MD_MEM_FLAG_FIXED) ?  *address_p : NULL;
```

Comment:
IMHO it's better to pass 'flags' down to uct_mm_md_mapper_ops(md)->alloc() routine, than to use address!=NULL as indication